src/device_builder.rs

```diff
@@ -106,27 +107,60 @@ impl<'a, B: UsbBus> UsbDeviceBuilder<'a, B> {
         self
     }
 
+    /// Sets **extra** Language ID for device.
+    ///
+    /// Since "en_US"(0x0409) is implicitly embedded, you just need to fill other LangIDs
+    ///
+    /// Default: (none)
+    pub fn extra_lang_ids(mut self, extra_lang_ids: &'a [u16]) -> Self {
+        assert!(
+            extra_lang_ids.len() < 16,
+            "Not support more than 15 extra LangIDs"
+        );
+
+        if extra_lang_ids.len() > 0 {
+            self.config.extra_lang_ids = Some(extra_lang_ids);
+        }
+
+        self
+    }
+
     /// Sets the manufacturer name string descriptor.
     ///
+    /// the first string should always be in English, the language of reset strings
+    /// should be pair with what inside [.extra_lang_ids()](Self::extra_lang_ids)
+    ///
     /// Default: (none)
-    pub fn manufacturer(mut self, manufacturer: &'a str) -> Self {
-        self.config.manufacturer = Some(manufacturer);
+    pub fn manufacturer(mut self, manufacturer_ls: &'a [&'a str]) -> Self {
+        if manufacturer_ls.len() > 0 {
+            self.config.manufacturer = Some(manufacturer_ls);
+        }
```

Comment:
Check that the number of lang strings provided is equal to extra-lang-ids + 1, throw an error otherwise.